### ISSUE TITLE
Installing via packagist results in 'minimum-stability' error.

### ISSUE THREAD

**Issue by moxxuk** [matepaiva/wp-graphql-crb#3]
Following the 'Usage' in the readme, and installing the package using the following:

```composer require matepaiva/wp-graphql-crb```

...I am faced with the following:
```[InvalidArgumentException]
Could not find a version of package matepaiva/wp-graphql-crb matching your minimum-stability (stable). Require it with an explicit version constraint allowing its desired stability.
```

What is the recommended way to install?

**Comment by matepaiva** [comment]
The package didn't have a stable release ─ until now. I have just published one, called 0.0.1. So please uninstall the package and try installing it again, please. If it solves your problem, please let me know so I can close this issue, ok?

Thank you!

PS.: That said, you have to keep in mind that this package is far from being really stable, so you should test a lot to see if your usage is ok. Also, the contributions are very welcome.

**Comment by moxxuk** [comment]
Thanks @matepaiva - Yes, this is solved.

I was wondering whether giving it a stable release was the right thing to do - on the one hand, there's more work to do to ensure it's 100% stable, yet on the other hand, without a stable release you couldn't install it from packagist unless you knew what you were doing with `minimum-stability` etc in composer.json. The readme also clearly states that it is experimental, and really if it is to be on packagist, it should have a stable version.

So in conclusion, I think that was the right thing to do! Thanks again.
